README.md

```diff
@@ -23,6 +23,19 @@ Find a bug or want to request a new feature?  Please let us know by [submitting
 
 Anyone and everyone is welcome to contribute.
 
+## Contributors
+[![All Contributors](https://img.shields.io/badge/all_contributors-20-orange.svg?style=flat-square)](#contributors-)
+|||||
+|:--:|:--:|:--:|:--:|
+|<img src="https://avatars.githubusercontent.com/u/19823035?v=4&s=100" width=100></img></br>[Luke Smallwood](https://github.com/lsmallwood)|<img src="https://avatars.githubusercontent.com/u/5695392?v=4" width=100></img></br>[James Ballard](https://github.com/JamesMBallard)|<img src="https://avatars.githubusercontent.com/u/1651508?&v=4" width=100></img></br>[Guillaume Belz](https://github.com/GuillaumeBelz)|<img src="https://avatars.githubusercontent.com/u/4107363?v=4" width=100></img></br>[Lucas Danzinger](https://github.com/ldanzinger)|
+|<img src="https://avatars.githubusercontent.com/u/30637561?v=4" width=100></img></br>[Ryan Kim](https://github.com/ryankim923)|<img src="https://avatars.githubusercontent.com/u/3836983?v=4" width=100></img></br>[Thomas Dunn](https://github.com/tdunn)|<img src="https://avatars.githubusercontent.com/u/42203545?v=4" width=100></img></br>[Archibald Neil MacDonald](https://github.com/anmacdonald)|<img src="https://avatars.githubusercontent.com/u/4101783?v=4" width=100></img></br>[Siddesh Pillai](https://github.com/siddeshpillai)|<img src="https://avatars.githubusercontent.com/u/5704590?v=4" width=100></img></br>[Nandini](https://github.com/nandinirao)
```

Comment:
❌ Shobana Suresh is duplicated in this table.
❓ There are two Ryan Kim contributors in this repo. Could you just confirm that both Ryan Kim accounts contributed to this repo?

❓ Is this ordered in any way? Should we order it? What's the commonly accepted way of doing this?

Reply:
It is not ordered in any way, could do by name if we wish.
Ryan Kim has pushed with two different profiles

Reply:
But I would not advocate for a sorting. This is because at the moment it is quite a manual labor to build the table, in case a new contributor has to be added there is a bit of work to shift people around to make space for it in the right place. Might be better to wait to sort once bot is set up or I write a better formula to create the table!